packages/backend-core/src/middleware/contentSecurityPolicy.ts

```diff
@@ -87,27 +89,45 @@ const CSP_DIRECTIVES = {
 }
 
 export async function contentSecurityPolicy(ctx: any, next: any) {
```

Comment:
While you're here, can you fix these anys?

```suggestion
import { Ctx } from "@budibase/types"
import { Next } from "koa"
export async function contentSecurityPolicy(ctx: Ctx, next: Next) {
```

Reply:
I can indeed, no problemo.

Reply:
Actually I can't type `next` as backend-core doesn't have `koa` as a dependency. So it's either leave it as `any` or add koa as a dependency. Happy to do either. Or move this middleware over to the server I suppose. Any preference?

Reply:
It already depends on some other koa packages, so I've just added in koa itself too which lets me type everything properly.